tests/patient-registration.spec.ts

```diff
@@ -115,8 +121,9 @@ test.describe("Patient Registration", () => {
       await stateCombobox.click();
 
       // Select the state option by visible text
+      await page.waitForTimeout(1000); // Allow time for dropdown to fully render
       const stateOption = page.getByRole("option", { name: patientData.state });
-      await stateOption.waitFor({ state: "visible", timeout: 5000 });
+      await stateOption.waitFor({ state: "visible", timeout: 10000 });
```

Comment:
again, we should avoid explicit timeouts, instead check for the presence of something.